arch/M68K/M68KDisassembler.c

```diff
@@ -143,44 +152,44 @@
 
 ///////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
 
-unsigned int m68k_read_disassembler_16(m68k_info *info, const uint64_t address)
+static unsigned int m68k_read_disassembler_16(const m68k_info *info, const uint64_t address)
 {
 	const uint64_t addr = (address - info->baseAddress) & info->address_mask;
 	if (addr > (info->code_len - 2)) {
 		return 0xaaaa;
 	}
-	uint16_t v0 = info->code[addr + 0];
-	uint16_t v1 = info->code[addr + 1];
+	const uint16_t v0 = info->code[addr + 0];
```

Comment:
please move variable declarations to the beginning of code block (which is function in this case).
this issue should be fixed in other places of this PR as well.